worker/src/main/resources/application.properties

```diff
@@ -57,4 +57,7 @@ spring.autoconfigure.exclude[0]=org.springframework.boot.autoconfigure.security.
 ## -----------------------------------------------------------------------------------------------------  BFD Health Check
 bfd.health.check.schedule=*/15 * * * * ?
 bfd.health.check.consecutive.successes=5
-bfd.health.check.consecutive.failures=5
\ No newline at end of file
+bfd.health.check.consecutive.failures=5
+
+eventlogger.kinesis.stream=bfd-insights-ab2d-dummy
+eventlogger.kinesis.region=US_EAST_1
```

Comment:
Does worker really need these properties?

Reply:
I was getting an error, but that was probably just in the testing setup.